lib/locasms.rb

```diff
@@ -6,10 +6,12 @@
 autoload :Logger,     'logger'
 
 module LocaSMS
-  autoload :Client,     'locasms/client'
-  autoload :Exception,  'locasms/exception'
-  autoload :Numbers,    'locasms/numbers'
-  autoload :RestClient, 'locasms/rest_client'
+  autoload :Client,            'locasms/client'
+  autoload :Exception,         'locasms/exception'
+  autoload :InvalidLogin,      'locasms/exception'
+  autoload :InvalidOperation,  'locasms/exception'
+  autoload :Numbers,           'locasms/numbers'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.